src/Symfony/Component/Messenger/Command/FailedMessagesRetryCommand.php

```diff
@@ -213,6 +216,10 @@ private function runWorker(string $failureTransportName, ReceiverInterface $rece
                 return;
             }
 
+            if ('skip' === $choice) {
+                $this->eventDispatcher->dispatch(new WorkerMessageSkipEvent($envelope, $envelope->last(SentToFailureTransportStamp::class)->getOriginalReceiverName()));
```

Comment:
shouldnt the code stop after that?
seeing the `$receiver->reject($envelope);` after seem weird

Reply:
also in the help() method above, you need to change the doc part of:

`The command will interactively ask if each message should be retried or discarded.`

`The command will interactively ask if each message should be retried, discarded or skipped.` :)